<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
<topic id="identity_alarmdefinitions">
  <title><ph conkeyref="HOS-conrefs/product-title"/>Identity Alarms</title>
  <body>
    <section>
      <p>These alarms show under the Identity section of the HPE Helion Ops Console.</p>
      <p>
        <table frame="all" rowsep="1" colsep="1" id="table_gt2_hzp_mx">
          <tgroup cols="5">
            <colspec colname="c1" colnum="1"/>
            <colspec colname="c2" colnum="2"/>
            <colspec colname="c3" colnum="3"/>
            <colspec colname="c4" colnum="4"/>
            <colspec colname="c5" colnum="5"/>
            <thead>
              <row>
                <entry>Service</entry>
                <entry>Alarm Name</entry>
                <entry>Description</entry>
                <entry>Likely Cause</entry>
                <entry>Mitigation Tasks to Perform</entry>
              </row>
            </thead>
            <tbody>
              <row>
                <entry morerows="4">identity-service</entry>
                <entry>HTTP Status</entry>
                <entry>
                  <codeblock>component=keystone-api
api_endpoint=public</codeblock>
                  <p>This check is contacting the Keystone public endpoint directly. </p>
                </entry>
                <entry>The Keystone service is down on the affected node.</entry>
                <entry>Restart the Keystone service on the affected node: <ol id="ol_ht2_hzp_mx">
                    <li>Log in to the lifecycle manager.</li>
                    <li>Use the Keystone start playbook against the affected node:
                      <codeblock>cd ~/scratch/ansible/next/hos/ansible
ansible-playbook -i hosts/verb_hosts keystone-start.yml --limit &lt;hostname></codeblock></li>
                  </ol></entry>
              </row>
              <row>
                <entry>HTTP Status</entry>
                <entry>
                  <codeblock>component=keystone-api
api_endpoint=admin</codeblock>
                  <p>This check is contacting the Keystone admin endpoint directly.</p>
                </entry>
                <entry>The Keystone service is down on the affected node.</entry>
                <entry>Restart the Keystone service on the affected node: <ol id="ol_it2_hzp_mx">
                    <li>Log in to the lifecycle manager.</li>
                    <li>Use the Keystone start playbook against the affected node:
                      <codeblock>cd ~/scratch/ansible/next/hos/ansible
ansible-playbook -i hosts/verb_hosts keystone-start.yml --limit &lt;hostname></codeblock></li>
                  </ol></entry>
              </row>
              <row>
                <entry>HTTP Status</entry>
                <entry>
                  <codeblock>component=keystone-api
monitored_host_type=vip</codeblock>
                  <p>This check is contacting the Keystone admin endpoint via the virtual IP address
                    (HAProxy).</p>
                </entry>
                <entry>The Keystone service is unreachable via the virtual IP address.</entry>
                <entry>If neither the <codeph>api_endpoint=public</codeph> or
                    <codeph>api_endpoint=admin</codeph> alarms are triggering at the same time then
                  there is likely a problem with haproxy. <p>You can restart the haproxy service
                    with these steps:</p>
                  <ol id="ol_jt2_hzp_mx">
                    <li>Log in to the lifecycle manager.</li>
                    <li>Use this playbook against the affected node:
                      <codeblock>cd ~/scratch/ansible/next/hos/ansible
ansible-playbook -i hosts/verb_hosts FND-CLU-start.yml --limit &lt;hostname></codeblock></li>
                  </ol></entry>
              </row>
              <row>
                <entry>Process Check</entry>
                <entry>Separate alarms for each of these Glance services, specified by the
                    <codeph>component</codeph> dimension: <ul id="ul_kt2_hzp_mx">
                    <li>keystone-main</li>
                    <li>keystone admin</li>
                  </ul></entry>
                <entry>Process crashed.</entry>
                <entry>
                  <p>You can restart the Keystone service with these steps:</p>
                  <ol id="ol_lt2_hzp_mx">
                    <li>Log in to the lifecycle manager.</li>
                    <li>Use this playbook against the affected node:
                      <codeblock>cd ~/scratch/ansible/next/hos/ansible
ansible-playbook -i hosts/verb_hosts keystone-start.yml --limit &lt;hostname></codeblock></li>
                  </ol>
                  <p>Review the logs in <codeph>/var/log/keystone</codeph> on the affected node.</p>
                </entry>
              </row>
              <row>
                <entry>Service Log Directory Size</entry>
                <entry>Service log directory consuming more disk than its quota.</entry>
                <entry>This could be due to a service set to <codeph>DEBUG</codeph> instead of
                    <codeph>INFO</codeph> level. Another reason could be due to a repeating error
                  message filling up the log files. Finally, it could be due to log rotate not
                  configured properly so old log files are not being deleted properly.</entry>
                <entry>Find the service that is consuming too much disk space. Look at the logs. If
                    <codeph>DEBUG</codeph> log entries exist, set the logging level to
                    <codeph>INFO</codeph>. If the logs are repeatedly logging an error message, do
                  what is needed to resolve the error. If old log files exist, configure log rotate
                  to remove them. You could also choose to remove old log files by hand after
                  backing them up if needed.</entry>
              </row>
            </tbody>
          </tgroup>
        </table>
      </p>
    </section>
  </body>
</topic>
